app/views/account/weeklyRenew.scala.html

```diff
@@ -62,7 +63,7 @@ <h3 class="mma-section__header">Renew your subscription</h3>
                 data-billing-country="@billToCountry.alpha2"
                 data-promo-code="@promoCode.map(_.get)"
                 data-currency="@currency.iso"
-                data-delivery-country="@contact.mailingCountryParsed.map(_.alpha2).getOrElse(billToCountry.alpha2)"
+                data-delivery-country="@contact.country.map(_.alpha2).getOrElse(billToCountry.alpha2)"
```

Comment:
We should consider making the SoldToContact.country non-optional given that it's a required field in our Zuora tennant (because we have ZTax installed): https://knowledgecenter.zuora.com/DC_Developers/G_SOAP_API/E1_SOAP_API_Object_Reference/Contact